packages/edit-site/src/components/editor/index.js

```diff
@@ -57,7 +57,9 @@ function Editor() {
 		page,
 		template,
 		select,
+		hasDirtyEntityRecords,
 	} = useSelect( ( _select ) => {
+		const { __experimentalGetDirtyEntityRecords } = _select( 'core' );
```

Comment:
I actually wonder if this same check should be used in the edit-post as well. You can edit site title there if you FSE is enabled. 

So potentially bundle the check back into `UnsavedChangesWarning`

Reply:
Ah, interesting. I was wondering if `__experimentalGetDirtyEntityRecords` could work in that context as well, but I'm not familiar enough with it to tell if it would have unwanted side effects.

If we change the warning component that way, we should probably also modify the Save button accordingly?

Reply:
Wanna give that a stab @david-szabo97 ? :smile:

Reply:
>  If we change the warning component that way, we should probably also modify the Save button accordingly?

I think the "save" button already triggers the multi-entity flow in edit-post when multiple entities are edited. That said, it's not "stable" because there's no stable block that can trigger a multi-entity flow yet.

Reply:
Yep, that's true. the save flow looks exactly the same as in site editor if there are multiple affected entities

Reply:
Done in https://github.com/WordPress/gutenberg/pull/24719.